RetakesAllocator/RetakesAllocator.cs

```diff
@@ -73,9 +75,25 @@ public override void Unload(bool hotReload)
     [CommandHelper(whoCanExecute: CommandUsage.CLIENT_ONLY)]
     public void OnGunsCommand(CCSPlayerController? player, CommandInfo commandInfo)
     {
-        commandInfo.ReplyToCommand(
-            $"{MessagePrefix}Eventually this will be a weapon menu. For now, please use !gun <weapon>."
-        );
+        HandleGunsCommand(player, commandInfo);
+    }
+    
+    private void HandleGunsCommand(CCSPlayerController? player, CommandInfo commandInfo)
+    {
+        if (player == null || !player.IsValid)
```

Comment:
I have `Helpers.PlayerIsValid`

Reply:
Have addressed that, it's annoying because C# doesn't seem to understand that **my helper is ensuring this value is not null**

Reply:
yaaa. Ive been abusing `?` most places since I rarely need a non-null reference until the very end, and then I just do a null check on the very last thing. Too bad it doesn't have type guards like typescript :(